changes-set.txt

```diff
@@ -60,7 +60,7 @@ proposed  cbvopab1  cbvopab1v
 proposed  syl       imtri       (analogous to *bitr*, sstri, etc.)
                                 there is less agreement on renaming syl
                                 than others here
```

Comment:
"there is less agreement on renaming syl than others here" -> this also applies to syld then...